src/frontend/app/shared/shared.module.ts

```diff
@@ -157,11 +154,11 @@ import { ValuesPipe } from './pipes/values.pipe';
     NestedTabsComponent,
     CardCfOrgUsageComponent,
     CardCfOrgUserDetailsComponent,
-    AppChipsComponent,
-    CfSpacePermissionCellComponent,
     CardCfSpaceUsageComponent,
     CardCfSpaceDetailsComponent,
-    TableCellRouteAppsAttachedComponent,
+    AppChipsComponent,
+    CfSpacePermissionCellComponent,
+    TableCellRouteAppsAttachedComponent
```

Comment:
Both of these entries shouldn't be needed (they'll be included in listCardComponents and listTableComponents above)